### ISSUE TITLE
Changes without "committing" or "pushing"

### ISSUE THREAD

**Issue by ghost** [susanBuck/e2-fall22#39]
Hi all,

It may be a silly question.

How the changes are happening without even "committing" or "pushing" the changes to the server?

Thank you

**Comment by susanBuck** [comment]
Hi @2015cambridge - All the changes you make are happening directly on the server. This is the essence of remote development - you are developing remotely on a server.

This is different than a workflow where you might make code changes to your files on your computer and then push those changes to the server.

Does that make sense?

**Comment by ghost** [comment]
Thank you for replying.

It's funny! I had to constantly commit and push changes to see the changes on the server. Is that mean that the commits & pushes are just to keep the GitHub repository updated? If yes, then I was missing an important part, and I am trying to figure out how I missed this. Thank you again

How long after the class should we access the course materials

**Comment by susanBuck** [comment]
Hi @2015cambridge - 

Correct - with how we've set things up, the commits/pushes are just to keep the Github repository updated. 

That's how in lecture videos I consistently make changes to code files and see the results in the browser instantly. I only do commits/pushes at notable check in points - e.g. when I've demonstrated building a project and talked about getting it ready for submission.

Hope that clears up any confusion : )

RE: Course material after the semester ends: 
https://hesweb.dev/e2/notes/misc/notes-about#faq-will-i-have-access-to-these-notes-after-this-semester
